client/payment-details/summary/index.tsx

```diff
@@ -229,7 +243,7 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 							{ balance.refunded ? (
 								<p>
 									{ `${ __(
-										'Refunded',
+										'Deducted',
```

Comment:
⚠️ This should be renamed `Deducted` for disputed transactions only – not for refunded transactions.

Reply:
Addressed in ee89414

Reply:
Added explicit tests for this logic in f0b9f52.